src/Core/Parser/SyntaxTree/Expression/NullcoalescingExpressionNode.php

```diff
@@ -0,0 +1,85 @@
+<?php
+namespace TYPO3Fluid\Fluid\Core\Parser\SyntaxTree\Expression;
+
+/*
+ * This file belongs to the package "TYPO3 Fluid".
+ * See LICENSE.txt that was shipped with this package.
+ */
+
+use TYPO3Fluid\Fluid\Core\Compiler\TemplateCompiler;
+use TYPO3Fluid\Fluid\Core\Parser\BooleanParser;
+use TYPO3Fluid\Fluid\Core\Parser\SyntaxTree\BooleanNode;
+use TYPO3Fluid\Fluid\Core\Rendering\RenderingContextInterface;
+use TYPO3Fluid\Fluid\Core\Variables\VariableExtractor;
+
+/**
+ * Ternary Condition Node - allows the shorthand version
```

Comment:
@CDRO, the comments in this file still refer to the ternary operator...

Reply:
Thanks for pointing this out, I've corrected this now.